testsuite/cluster-test/src/deployment.rs

```diff
@@ -23,18 +23,14 @@ pub struct DeploymentManager {
 }
 
 const VALIDATOR_IMAGE_REPO: &str = "libra_e2e";
-const CLIENT_IMAGE_REPO: &str = "libra_client";
-const FAUCET_IMAGE_REPO: &str = "libra_faucet";
-pub const SOURCE_TAG: &str = "nightly";
-pub const TESTED_TAG: &str = "nightly_tested";
+const NIGHTLY_PREFIX: &str = "nightly";
 const UPSTREAM_PREFIX: &str = "upstream_";
```

Comment:
I supposed this is for the transition period and we will clean up the upstream_xxx remnants later?